aptos-move/e2e-move-tests/src/lib.rs

```diff
@@ -37,3 +41,14 @@ impl PackageHooks for AptosPackageHooks {
         bail!("not used")
     }
 }
+
+pub(crate) fn build_package(
+    package_path: PathBuf,
+    options: BuildOptions,
+) -> anyhow::Result<BuiltPackage> {
+    let mut options = options;
+    if read_env_var(ENABLE_V2) == "1" {
+        options.compiler_version = Some(CompilerVersion::V2);
```

Comment:
Apparently this line is not covered by tests.  Can we find out how the rest of this file is being covered by tests and add a similar test that runs with `ENABLE_V2=1` in the environment?

Reply:
I tested manually on my machine by setting the env var ENABLE_V2=1 and the number of tests being run are identical. Not sure whether it is enough?

Reply:
Since this flag is not enabled in CI it can't be tested this way. Manual is good enough I'd say.